scripts/nb2doc/asciidoc-template/index.adoc.j2

```diff
@@ -0,0 +1,94 @@
+// DO NOT EDIT - AsciiDoc file generated automatically
```

Comment:
Is this based on a template from somewhere?

Wonder if f.i. we use Latex even

Reply:
Yes, this is based on the base Asciidoc template included in `nbconvert`. I've kept everything and only added to it or made changes were I saw fit, but we can customise it as we want.